detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md

```diff
@@ -66,6 +66,13 @@
   * devDependencies
     * @opentelemetry/contrib-test-utils bumped from ^0.35.1 to ^0.36.0
 
```

Comment:
What's going on with all the `### Dependencies` sections above here?

I see it in auto-instrumentations-node/CHANGELOG.md below as well.

It's in these files in this repo:

```
% git ls-files | rg CHANGELOG.md | while read f; do if [[ $(cat $f | head -3 | tail -1) == "### Dependencies" ]]; then echo $f; fi ; done
detectors/node/opentelemetry-resource-detector-alibaba-cloud/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-azure/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-container/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-gcp/CHANGELOG.md
detectors/node/opentelemetry-resource-detector-instana/CHANGELOG.md
metapackages/auto-instrumentations-node/CHANGELOG.md
metapackages/auto-instrumentations-web/CHANGELOG.md
packages/opentelemetry-propagation-utils/CHANGELOG.md
plugins/node/instrumentation-amqplib/CHANGELOG.md
plugins/node/instrumentation-lru-memoizer/CHANGELOG.md
plugins/node/instrumentation-mongoose/CHANGELOG.md
plugins/node/instrumentation-socket.io/CHANGELOG.md
plugins/node/instrumentation-tedious/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-cassandra/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-memcached/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mongodb/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mysql/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-mysql2/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-pg/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-redis-4/CHANGELOG.md
plugins/node/opentelemetry-instrumentation-redis/CHANGELOG.md
```

Going back looking at changes to `detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md` to see when a `### Dependencies` section was incorrectly added to the top (not under an h2), the last good PR was:

    chore: release main (#1590)

then starting with this release PR it started incorrectly adding that section:

    chore: release main (#1683)

and continued for a number of releases:

    ...
    chore: release main (#1890)

until fixed by the time this release PR was made:

    chore: release main (#1935)


Anyway... this is independent of this PR. I'll open a separate issue.

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2207 opened for this.